src/main/kotlin/apply/ui/api/EvaluationRestController.kt

```diff
@@ -0,0 +1,53 @@
+package apply.ui.api
+
+import apply.application.EvaluationData
+import apply.application.EvaluationResponse
+import apply.application.EvaluationService
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/api/recruitments/{recruitmentId}/evaluations")
+class EvaluationRestController(
+    private val evaluationService: EvaluationService,
+) {
+
+    @PostMapping
+    fun createEvaluation(
+        @PathVariable("recruitmentId") recruitmentId: Long,
+        @RequestBody evaluationData: EvaluationData,
+    ): ResponseEntity<Unit> {
+        evaluationService.save(evaluationData)
+        return ResponseEntity.ok().body(Unit)
+    }
+
+    @GetMapping("{evaluationId}")
```

Comment:
```suggestion
    @GetMapping("/{evaluationId}")
```
`/`가 누락된 것 같은데 혹시 누락되어도 매핑이 잘 될까요?!ㅎㅎㅎ

Reply:
헉.....👀 바로 고쳤습니다!! 
Rest는 일단 keep하기로 한 것 같지만... 🤔이야기가 다시 나온김에 `Rest`는 지웠습니다. 😃